### PR TITLE
Added Shell Autocompletion feature in Installation guide

### LINKED ISSUE #386: Add link to bash/zsh autocompletion in installation guide
difficulty newcomer

Some sentence and a link to the page which explains how to setup autocompletion would be helping users looking for that.

### PR DESCRIPTION
**Pull request by vedantrathore**
This feature fixes #386

### CODE REVIEW

File: Users/Install.rst [hunk 1]
```diff
@@ -280,6 +280,14 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+
+Integrating Shell Autocompletion
+--------------------------------
+
+You can also use coala for tab completion in bash or zsh. This will autocomplete coala arguments and bear names.
+You can set them up `here <http://coala.readthedocs.io/en/stable/Users/Tutorials/Shell_Autocompletion.html>`_.
+
```
Comment (hemangsk) [inline Users/Install.rst]: L287 & L288 have length ~ 112, so you try moving stuff from to new lines so that it stays in length limit ie (79)
😃  👍

Comment (vedantrathore) [inline Users/Install.rst]: Okk sure, thanks!

**Comment by hemangsk** [comment]
Hey first off you're in the absolutely right direction 👍 , you just need to run coala on your code, and it'll solve all the GitMate issues arising here,  (just ```cd``` into the repo's root and run ```$ coala``` )


Apart from that the important stuff to keep in mind is _coala commit guidelines_, which you can find here [coala.io/commit](http://coala.io/commit)


And in case of any queries just shoot'em at Gitter or here 👍  😃

File: Users/Install.rst [hunk 2]
```diff
@@ -280,6 +280,13 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+Integrating Shell Autocompletion
+--------------------------------
+
+You can also use coala for tab completion in bash or zsh.
+This will autocomplete coala arguments and bear names.
+You can set them up `here <http://coala.readthedocs.io/en/stable/Users/Tutorials/Shell_Autocompletion.html>`_.
```
Comment (Mariatta) [inline Users/Install.rst]: Tutorials/Shell_Autocompletion is an internal documentation that can be cross-referenced using sphinx role :doc: as described here: http://www.sphinx-doc.org/en/1.5.1/markup/inline.html#role-doc

Can you try doing that, instead of using an external link? 

( http://coala.readthedocs.io/en/stable/Users/Tutorials/Shell_Autocompletion.html is considered an external link )

Perhaps also rephrase this to:
```
You can set them up by following this tutorial. (tutorial as hyperlink)
```

After that, please use squash all of your commits into one.
Instructions for squashing commits are here:
http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

Thanks.

**Review by Mariatta — CHANGES_REQUESTED** [review]
1. Adjust the url.
2. Squash the commits.

File: Users/Install.rst (hunk 2 above)
Comment (vedantrathore) [inline Users/Install.rst]: Thanks @Mariatta and @hemangsk I'll look into it

**Comment by Adrianzatreanu** [comment]
@vedantrathore hey you still working on this :) ? ping us on coala.io/chat if you need any kind of help :D
